front/app/containers/Admin/pagesAndMenu/containers/CustomPages/Edit/HeroBanner/index.tsx

```diff
@@ -152,6 +194,32 @@ const EditCustomPageHeroBannerForm = ({
         <HelmetIntl title={messages.customPageMetaTitle} />
         <GenericHeroBannerForm
           onSave={handleSave}
+          onSaveAndEnable={
+            // undefined to match type for optional prop.
+            // only show secondary button if banner is not enabled
+            localSettings.banner_enabled ? undefined : handleSaveAndEnable
+          }
+          badge={
+            localSettings.banner_enabled ? (
+              <StyledStatusLabel
+                text={
+                  <span style={{ color: colors.success }}>
+                    <FormattedMessage {...messages.heroBannerShown} />
```

Comment:
This one has a bullet in front (design). Would add this to the copy for RTL/Arabic if we add it.

Reply:
Looks like there are variants, I'll mess with it

Reply:
Yeah, not worth spending loads of time on. Thought it would be a 10 seconds change